ddtrace/contrib/pynamodb/patch.py

```diff
@@ -0,0 +1,80 @@
+"""
+Trace queries to botocore api done via a pynamodb client
+"""
+
+import os
+from ddtrace.vendor import wrapt
+from ddtrace import config
+import pynamodb.connection.base
+
+
+from ...constants import ANALYTICS_SAMPLE_RATE_KEY, SPAN_MEASURED_KEY
+from ...pin import Pin
+from ...ext import SpanTypes
+from ...utils.formats import deep_getattr
+from ...utils.wrappers import unwrap
```

Comment:
```suggestion
from ...utils.wrappers import unwrap
from .. import trace_utils
```

you'll need to merge master for this one

Reply:
I added this in the latest commit! Let me know if this looks fine now.